agent/util-scripts/pbench-register-tool-trigger

```diff
@@ -69,13 +77,18 @@ while true; do
 	esac
 done
 
-if [ "$start_trigger" != "${start_trigger%%:*}" ]; then
+if [[ -z "$start_trigger" && -z "$stop_trigger" ]] ;then
+	printf "At least one of --start-trigger or --stop-trigger must be specified.\n"
+	usage
+	exit 3
+fi
+if [[ "$start_trigger" != "${start_trigger%%:*}" ]]; then
 	printf "$script_name: the start trigger cannot have a colon in it: \"${start_trigger}\"\n"
```

Comment:
We should also probably ensure that the trigger does not contain a newline, as that will screw up the `tool-triggers` file.

Reply:
Agreed, though for the `0.69` branch I am not included to tackle this work, but instead fix this in a rewrite in Python3.